Add html to plain text note in storage guide (#9964)
supabase/supabase: apps/reference/docs/guides/storage.mdx

### Files

> Files can be any sort of media file. This includes images, GIFs, and videos. It is best practice to store files outside of your database because of their sizes.  HTML files will be returned as plain text for security reasons.

Reviewer: ```suggestion
Files can be any sort of media file. This includes images, GIFs, and videos. It is best practice to store files outside of your database because of their sizes. For security, HTML files are returned as plain text.
```
